tests/foreman/maintain/test_upgrade.py

```diff
@@ -146,11 +147,15 @@ def test_negative_pre_upgrade_tuning_profile_check(request, custom_host):
     )
     custom_host.download_repofile(product='satellite', release=last_y_stream)
     custom_host.execute('dnf -y module enable satellite:el8 && dnf -y install satellite')
-    # Install without system checks to get around installer checks
+    # Install with development tuning profile to get around installer checks
     custom_host.execute(
-        f'satellite-installer --scenario satellite --disable-system-checks --tuning {profile}',
+        'satellite-installer --scenario satellite --tuning development',
         timeout='30m',
     )
+    # Change to correct tuning profile (default or medium)
+    custom_host.execute(
+        f'sed -i "s/tuning: development/tuning: {profile}/g" {INSTALLER_CONFIG_FILE}'
+    )
```

Comment:
Shouldn't we rerun installer after tuning profile in INSTALLER_CONFIG_FILE? I don't think it'll make any sense for pre-upgrade checks to run if we don't run installer?

Reply:
I think the point of this test is to check whether the pre-upgrade checks catch the discrepancy. The question is, do the checks really use the config file? I'd suppose they do even though it's not specified explicitly. Otherwise, I think this test would fail on the last assert.

Also, this is just a cherrypick.

Reply:
yeah, but in reality this should be an main PR and cherrypick got merged https://github.com/SatelliteQE/robottelo/pull/12535, and main PR is called cherrypick here, lol :D

Reply:
yeah, cause I wanted to test the fix, which I can't do with master branch, so I raised it first against 6.14 🤷

Reply:
and tbh I don't like the current approach of us doing TFA for 6.14 and raising a fix against master branch and then running into a bunch of the issues that we never faced in automation cause we never ran the test with stream :|

Reply:
I don't think it's required to run installer, and like Lukas mentioned the assertion would fail.

Reply:
But if we run installer, I think it'll fail.